libsolidity/formal/SMTChecker.cpp

```diff
@@ -261,14 +262,14 @@ void SMTChecker::checkUnderOverflow(smt::Expression _value, IntegerType const& _
 	checkCondition(
 		_value < minValue(_type),
 		_location,
-		"Underflow (resulting value less than " + formatNumber(_type.minValue()) + ")",
+		"Underflow (resulting value less than " + (this->hexReadable == false ? formatNumber(_type.minValue()) : formatNumberReadable(_type.minValue())) + ")",
```

Comment:
I think in general we don't need a flag for this.

Reply:
I've fixed this since readable function doesn't apply to an underflow.